send/o365_mu_users_export_process.pl

```diff
@@ -86,7 +103,6 @@
 	if($loginsInTable->{$uco}) {
 		$skipped++;
 	} else {
-		my $insertLogin = $dbh->prepare(qq{INSERT INTO $tableName (uco) VALUES (?)});
 		$insertLogin->execute($uco);
```

Comment:
now this is a problem because `validLogins` contains users who have nonempty mfaStatus but should not be present in the O365 table